src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -21,6 +21,21 @@
             <i class="fa fa-external-link"></i>
           </a>
         </dd>
+        <ng-container>
+          <dt>Telemetry Dashboard
+            <cd-label
+              value="{{ telemetryEnabled ? 'Enabled' : 'Disabled' }}"
```

Comment:
Not sure if this makes sense but we call the telemetry enabled operation as Activate Telemetry!

```suggestion
              value="{{ telemetryEnabled ? 'Active' : 'Inactive' }}"
```

Reply:
Thanks i'll change it, With Active and Deactive, the color scheme is weird. Active is Yellow and Deactive is green. Is there a way to fix the colors with cd-label component?